common/src/constants.rs

```diff
@@ -38,3 +38,9 @@ pub mod limits {
     pub const MAX_PROJECTS_DEFAULT: u32 = 3;
     pub const MAX_PROJECTS_EXTRA: u32 = 15;
 }
+
+pub mod stripe_price_ids {
+    /// The price ID of the recurring AWS RDS instance product.
+    /// TODO: this is set to the id of the test product, set to real product before releasing.
+    pub const AWS_RDS_INSTANCE_RECURRING: &str = "price_1OIS06FrN7EDaGOjaV0GXD7P";
```

Comment:
Maybe this should rather be passed as an arg to auth?

Reply:
Implemented in latest commits.